### DISCUSSION TITLE
Is there any way to get varchar length limit??

### DISCUSSION THREAD

**Discussion by itezjaehyeon** [prisma/prisma#13403] [Q&A]
```
model Introduction {
  id       Int    @id @default(autoincrement())
  resumeId Int    @map("resume_id")
  title    String @db.VarChar(200)
  content  String @db.LongText
  resume   Resume @relation(fields: [resumeId], references: [id])

  @@index([resumeId], name: "FK_introduction_resume_id_resume_id")
  @@map("introduction")
}
```
Is there any way to get length limit of title (200)?? i found on Prisma.dmmf, etc..... but i couldn't find what i want...
anyone please help...

**Comment by nurul3101** [comment]
Hey @itezjaehyeon 👋 
So do you mean you want to get only the value "200" which is defined in the title column in the introduction model of the schema file?

**Reply by vladi-stevanovic** [reply]
👋 Hello @itezjaehyeon  I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by AzatCoder** [comment]
Is there any news? I'm trying to implement SSOT as much as I can, so this feature will be very helpful for me.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
